homeassistant/helpers/config_validation.py

```diff
@@ -871,6 +871,22 @@ def __call__(self, selected: list) -> list:
         return selected
 
 
+class expandable:
```

Comment:
I think we should either name this "section", or rename section to "expandable" in https://github.com/home-assistant/core/pull/116100 so we use the same language

Reply:
In the frontend this selector is called `expandable` for a while now, but I guess we can call it `section` here and keep the selector named `expandable`? Or should that be aligned?

Reply:
To aid core developers, we should use the same naming across blueprints, data flows and services.
I think `section` is good, `expandable` is how we've chosen to display the sections in frontend?

Reply:
Agree 👍